src/dbus_api/pool.rs

```diff
@@ -6,14 +6,15 @@ use std::{collections::HashMap, path::Path, vec::Vec};
 
 use dbus::{
     self,
-    arg::{Array, IterAppend},
+    arg::{Array, IterAppend, RefArg, Variant},
     tree::{
         Access, EmitsChangedSignal, Factory, MTFn, MethodErr, MethodInfo, MethodResult, PropInfo,
+        Tree,
     },
     Message,
 };
-
 use devicemapper::Sectors;
+use itertools::Itertools;
```

Comment:
We have a special ordering for imports, first from standard library, then external crates furnished by others, then external crates furnished by ourselves, then local imports. So itertools import should be above devicemapper import.